OpenEdXMobile/res/drawable/course_tabs_background.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item>
+        <shape android:shape="rectangle">
+            <solid android:color="@color/neutralLight" />
+        </shape>
+    </item>
+    <item android:bottom="@dimen/edx_x_quarter_margin">
+        <shape android:shape="rectangle">
+            <solid android:color="@color/toolbar_background_color" />
+        </shape>
+    </item>
+</layer-list>
```

Comment:
why needed this `layer-list`?

Reply:
To add white tabs background with grayish bottom border as unselected indicator.